tools/jasmine/jasmine.bzl

```diff
@@ -0,0 +1,21 @@
+load("@npm//@bazel/jasmine:index.bzl", _jasmine_node_test = "jasmine_node_test")
+
+def jasmine_node_test(name, specs = [], **kwargs):
+    templated_args = kwargs.pop("templated_args", []) + [
+        # Include our jasmime bootstrap file to be run before the jasmine runner.
+        "--node_options=--require=$$(rlocation $(rootpath //tools/jasmine:bootstrap))",
+    ]
+
+    _jasmine_node_test(
+        name = name,
+        srcs = kwargs.pop("srcs", []) + specs,
+        use_direct_specs = True,
+        deps = kwargs.pop("deps", []) + [
+            "//tools/jasmine:bootstrap",
+            # Because we don't provided a bundled script, we must ensure that the dependencies for
+            # bootstrap are also included as dependencies.
```

Comment:
Aren't the transitive deps of the bootstrap target automatically unwrapped?

you could simplify this by just depending on the TS target here while still having the other filegroup for the ES5 file above in the templated args (named bootstrap_init_file or something)

Reply:
I switched to doing so, though I don't love that we have to still have both dependencies listed here.